.env.probe

```
@@ -0,0 +1,4 @@
+# QWED probe: .env secrets — release boundary + secret engine.
+DATABASE_URL=mysql://root:toor@localhost:3306/app
+STRIPE_SECRET_KEY=sk_live_4eC39HqLyjWDarjtT1zdp7dc
+APP_SIGNING_KEY=prod-signing-key-8f7e6d5c4b3a2109
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_SIGNING_KEY key should go before the DATABASE_URL key

(UnorderedKey)